src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -91,7 +98,20 @@ public TlsDetails create(final SSLEngine sslEngine) {
     }
 
     /**
-     * Close this client.
+     * Creates High Level Rest Client for SDK.
+     * @param hostAddress The address of OpenSearch cluster, client can connect to
+     * @param port The port of OpenSearch cluster
+     * @return SDKClient which is internally an RestHighLevelClient. The user is responsible for calling {@link #doCloseRestClient()} when finished with the client
+     */
+    public RestHighLevelClient initializeRestClient(String hostAddress, int port) {
+        RestClientBuilder builder = builder(hostAddress, port);
+
+        highLevelClient = new RestHighLevelClient(builder);
+        return highLevelClient;
+    }
+
+    /**
+     * Close java client.
```

Comment:
Would it be a good idea for the SDKClient itself to implement `Closeable` and include a `close()` method that calls both of these methods?   Then we only have to do one single `close()` call for SDKClient.

Reply:
I'm not saying to remove the individual close, but to have a single call that attempts to close both (and is a no-op if uninitialized.).  Each of these individual ones already does a null check so are safe to call.

Reply:
Ah I see your ask. That's a good point. Pushed the change. Let me know if that's right